pkg/logs/client.go

```diff
@@ -74,24 +82,58 @@ func (c *Client) PrintComponent(route string, service string, skip int, filters
 		return fmt.Errorf("HTTP %d error", resp.StatusCode)
 	}
 	for scanner := bufio.NewScanner(resp.Body); scanner.Scan(); {
-		err := c.printEntry(scanner.Bytes())
+		err := c.printEntry(scanner.Bytes(), opts)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func (c *Client) printEntry(rawEntry []byte) error {
+func (c *Client) printEntry(rawEntry []byte, opts Options) error {
 	var entry Entry
 	err := json.Unmarshal(rawEntry, &entry)
 	if err != nil {
 		return err
 	}
 
+	enc := json.NewEncoder(c.out)
+	switch opts.Format {
+	case "json-pretty":
+		enc.SetIndent("", "    ")
+		fallthrough
+	case "json":
+		return enc.Encode(entry.Fields)
+	case "cat":
+		c.colorPrefix(entry.Fields.Priority)
+		fmt.Fprint(c.out, entry.Fields.Message)
+		c.colorPostfix()
```

Comment:
Did you mean `colorSuffix`? Also, this function is actually resetting the attributes, maybe it can be called `ansiReset` or `resetColor`.

As a side note, the way we're dealing with colors here is very manual, in the future we should consider having a more intuitive color API / library to use in the CLI.